### ISSUE TITLE
Compiler warnings

### ISSUE THREAD

**Issue by f4exb** [f4exb/serialDV#9]
```
 => => # /opt/build/serialDV/dvcontroller.cpp:201:18: warning: comparison is always false due to limited range of data type [-Wtype-limits]                                                                                                  
 => => #      if (dBGainIn < -90) {                                                                                                                                                                                                          
 => => #          ~~~~~~~~~^~~~~                                                                                                                                                                                                             
 => => # /opt/build/serialDV/dvcontroller.cpp:207:19: warning: comparison is always false due to limited range of data type [-Wtype-limits]                                                                                                  
 => => #      if (dBGainOut < -90) {    
```

**Comment by f4exb** [comment]
This happens on arm. In some systems `char` is not signed by default. Using `signed char` in the definition should fix this.

**Comment by f4exb** [comment]
Fixed in v1.1.2
